svgnative/src/ports/cairo/CairoSVGRenderer.cpp

```diff
@@ -122,7 +122,12 @@ void CairoSVGPath::CurveTo(float x1, float y1, float x2, float y2, float x3, flo
 
 void CairoSVGPath::CurveToV(float x2, float y2, float x3, float y3)
 {
-    cairo_curve_to(mPathCtx, mCurrentX, mCurrentY, x2, y2, x3, y3);
+    float cx1 = static_cast<float>(mCurrentX + 2.0 / 3.0 * (x2 - mCurrentX));
```

Comment:
`2.0f` and `3.0f`. After that, it should not be needed to cast to float.

Reply:
Fixed. Please check, thanks.